yb-voyager/cmd/root.go

```diff
@@ -32,14 +32,13 @@ var (
 	cfgFile    string
 	exportDir  string
 	startClean bool
-	logLevel   string
 	lockFile   lockfile.Lockfile
 )
 
 var rootCmd = &cobra.Command{
 	Use:   "yb-voyager",
-	Short: "A tool to migrate a database to YugabyteDB",
-	Long:  `Currently supports PostgreSQL, Oracle, MySQL. Soon support for DB2 and MSSQL will come`,
+	Short: "A CLI based migration engine to migrate complete database(schema + data) from some source database to YugabyteDB",
+	Long:  `A CLI based migration engine for complete database migration from a source database to YugabyteDB. Currently supported source databases are Oracle, MySQL, PostgreSQL.`,
```

Comment:
How about adding a link to the public documentation in the `Long` text?

Reply:
sure